docs/source/markdown/podman-image-mount.1.md

```diff
@@ -32,20 +32,21 @@ Print the mounted images in specified format (json).
 
 ## EXAMPLE
 
+Mount multiple images. In rootless mode, execute podman unshare first:
```

Comment:
The second part on `podman mount` may confuse users.  I find it unclear whether the user needs to do that or if podman does it automatically.  Maybe extend an example for an obvious rootless example?  Here it's clearly a root directory.

Reply:
Yes I hate the wording.   

How about?

Note: In rootless mode, image mounting works only after executing the `podman unshare` command to enter the user namespace.

Reply:
LGTM